lib/ex_json_schema/validator/error.ex

```diff
@@ -1,27 +1,180 @@
 defmodule ExJsonSchema.Validator.Error do
   defstruct [:error, :path]
 
+  alias ExJsonSchema.Validator.Error
+
+  @spec format(ExJsonSchema.Validator.errors) :: [{String.t, String.t}]
+  def format(errors) do
```

Comment:
I would probably be a good idea to have this return `:ok` when passing `:ok`. Just so it's always possible to do something like this:
```elixir
ExJsonSchema.Validator.validate(schema, %{"foo" => "bar"})
|> ExJsonSchema.Validator.Error.format
```

Reply:
will do